README.md

```diff
@@ -202,6 +207,15 @@ vim.keymap.set(
   - A debug adapter (e.g. [`lldb`](https://lldb.llvm.org/)
     or [`codelldb`](https://github.com/vadimcn/codelldb)
 
+  **NB!!!** make sure after install lldb ([`nvim-dap`](https://github.com/mfussenegger/nvim-dap) recommends [lldb-dap](https://lldb.llvm.org/resources/lldbdap.html))
```

Comment:
This isn't always necessary and this information belongs in the install instructions for lldb-dap.